howtos/how-to-create-a-popup-inbox/js/script.js

```diff
@@ -36,45 +36,21 @@ const currentUser = {
   role: "default",
 };
 
-const conversationItems = [
-  {
-    id: "conversation_0",
-    subject: "Chat with Bob The Builder",
-    user: {
-      id: "user_0",
-      name: "Bob The Builder",
-      email: ["bobthebuilder@email.com"],
-      photoUrl: "https://talkjs.com/images/avatar-2.jpg",
-      welcomeMessage: "Can we fix it? ⚒️",
-    },
-  },
-  {
-    id: "conversation_1",
-    subject: "Chat with Ash Ketchum",
-    user: {
-      id: "user_1",
-      name: "Ash Ketchum",
-      email: ["ashketchum@email.com"],
-      photoUrl: "https://talkjs.com/images/avatar-3.jpg",
-      welcomeMessage: "Gotta catch `em all 👾",
-    },
+const currentConversation = {
+  id: "conversation_0",
+  subject: "Chat with Bob The Builder",
+  user: {
+    id: "user_0",
+    name: "Bob The Builder",
+    email: ["bobthebuilder@email.com"],
+    photoUrl: "https://talkjs.com/images/avatar-2.jpg",
```

Comment:
Minor suggestion to avoid a console warning

```suggestion
    photoUrl: "https://talkjs.com/images/avatar-2.jpg",
    role: "default",
```